app/managers.py

```diff
@@ -0,0 +1,50 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("db/Cinema.db")
```

Comment:
Use `os.path` in the future to define paths that would work independently from directories delimiters on different operating systems